partials/tabs/people.html

```diff
@@ -35,3 +35,9 @@ <h4 class="no-margin light">{{ person.nick }}</h4>
   </div>
 </div>
 <link rel="stylesheet" href="/resources/css/people.css">
+<script>
+  $(document).ready(function(){
+    $('ul.tabs').tabs();
+  });
+
```

Comment:
Extra line, can be removed

Reply:
Same for all the files

Reply:
Not the newline at end of file, See L42 , L281 etc.

Reply:
done @hemangsk :+1:

Reply:
Sorry I think I wasn't clear enough, only those extra new lines which were added in this commit inside the script tags need to be omitted. Others are fine. If you feel they are extra then its a discussion for a new ticket and not this one.

So we are fine with just converting the
```
<script>
$(document).ready(function(){
 $('ul tabs').tabs();
})

``` 

to 

```
<script>
$(document).ready(function(){
 $('ul tabs').tabs();
})
```

Reply:
oh okay @hemangsk done :+1: